modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -938,6 +939,294 @@ m|WARNING
+Command::
++
+[source, cypher]
+----
+REVOKE ROLE reader FROM alice
+----
+
+Full title of the returned code::
+`REVOKE ROLE reader FROM alice` has no effect.
+
+Description of the returned code::
+User does not exist. Make sure nothing is misspelled.
+This notification will become an error in a future major version.
+See Status Codes documentation for more information.
+
+Suggestions for improvement::
+Verify that this is the intended user and that it is spelled correctly.
+====
+
+.Revoking a privilege from a non-existing role
+====
+Command::
++
+[source, cypher]
+----
+REVOKE GRANT WRITE ON GRAPH * FROM manager
+----
+
+Full title of the returned code::
+`REVOKE GRANT WRITE ON GRAPH * FROM manager` has no effect.
+
+Description of the returned code::
+Role does not exist. Make sure nothing is misspelled.
+This notification will become an error in a future major version.
+See Status Codes documentation for more information.
+
+Suggestions for improvement::
+Verify that this is the intended role and that it is spelled correctly.
+====
+
+.Revoking a privilege on a non-existing graph from a role
+====
+Command::
++
+[source, cypher]
+----
+REVOKE GRANT WRITE ON GRAPH neo3j FROM editor
+----
+
+Full title of the returned code::
+`REVOKE GRANT WRITE ON GRAPH neo3j FROM editor` has no effect.
+
+Description of the returned code::
+// We use 'Database' here for multiple reasons
+// - The user fixes the issue by doing 'create database', not 'create graph'.
+// - This matches the existing behaviour when granting the privilege.
+// - It is easier to implement this way.
+Database 'neo3j' does not exist. Make sure nothing is misspelled.
+This notification will become an error in a future major version.
+See Status Codes documentation for more information.
+
+Suggestions for improvement::
+Verify that this is the intended graph and that it is spelled correctly.
+====
+
+.Revoking a privilege on a non-existing database from a role
+====
+Command::
++
+[source, cypher]
+----
+REVOKE GRANT ACCESS ON DATABASE neo3j FROM editor
+----
+
+Full title of the returned code::
+`REVOKE GRANT ACCESS ON DATABASE neo3j FROM editor` has no effect.
+
+Description of the returned code::
+Database 'neo3j' does not exist. Make sure nothing is misspelled.
+This notification will become an error in a future major version.
+See Status Codes documentation for more information.
+
+Suggestions for improvement::
+Verify that this is the intended database and that it is spelled correctly.
+====
+
+.Revoking a privilege from a role with wildcard graph parameter
+====
+Parameter::
++
+[source, javascript]
+----
+{
+    "graph": "*"
+}
+----
+Command::
++
+[source, cypher]
+----
+REVOKE GRANT CREATE ON GRAPH $graph FROM PUBLIC
+----
+
+Full title of the returned code::
+`REVOKE GRANT CREATE ON GRAPH $graph FROM PUBLIC` has no effect.
+
+Description of the returned code::
+Parameterized database and graph names do not support wildcards. 
+Verify that everything is spelled correctly.
```

Comment:
I think this sentence have been updated by mistake, should be
```suggestion
Make sure nothing is misspelled.
```
to match the other ones (it's just the cause that differs)

Reply:
I _think_ writing positively is better when possible.

Reply:
Another factor is that we show description and suggestion for improvement next to each other in the docs, so we get repeated sentences. Using `Make sure` in description and `Verify that` in suggestion for improvement makes the text a little more natural.